src/controllers/loginController.js

```diff
@@ -0,0 +1,39 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import User from '../models/loginModel';
+
+const userLogin = (req, res) => {
+  User.find({ email: req.body.email }).exec().then((user) => {
+    if (user < 1) {
+      return res.status(500).json({
+        message: 'Auth failed',
+      });
+    }
+
+    bcrypt.compare(req.body.password, user[0].password, (err) => {
+      if (err) {
+        res.status(500).json({
+          message: 'Auth failed',
+        });
+      } else {
+        const token = jwt.sign({
+
+          email: req.body.email,
+          userId: user[0]._id,
+
+        },
+        process.env.TOKEN,
```

Comment:
Also in the root of your dir create a file called `.env.example` and keep updating it as you add more environment variables. Make sure that you only keep an updated list of all env vars you're using/adding to the project.  Keep just the keys in there, not the values. this means that this file will be a copy of .env but without the values.